.meta.toml

```diff
@@ -0,0 +1,40 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/c-code
+[meta]
+template = "c-code"
+commit-id = "17719b8b55c2f7f9bf278e62fc1ae263a70bd9b4"
+
+[python]
+with-appveyor = true
+with-windows = false
+with-pypy = true
+with-legacy-python = true
+with-docs = true
+with-sphinx-doctests = false
+
+[tox]
+use-flake8 = false
+
+[coverage]
+fail-under = 99
+
+[manifest]
+additional-rules = [
+    "include *.sh",
```

Comment:
To automatically add this I created https://github.com/zopefoundation/meta/pull/117.